src/components/Snackbar/AptoSnackbar.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
```

Comment:
Tests?

Reply:
it's on my todos I wanted to see what we thought of this before doing them